runtime/near-vm-runner/src/tests.rs

```diff
@@ -24,14 +26,27 @@ const PREDECESSOR_ACCOUNT_ID: &str = "carol";
 const LATEST_PROTOCOL_VERSION: ProtocolVersion = ProtocolVersion::MAX;
 
 fn with_vm_variants(runner: fn(VMKind) -> ()) {
+    #[cfg(feature = "wasmer0_vm")]
+    static WASMER0_TRAP_HANDLER_SETUP: OnceCell<()> = OnceCell::new();
+    #[cfg(feature = "wasmer0_vm")]
+    WASMER0_TRAP_HANDLER_SETUP.get_or_init(|| {
```

Comment:
A more minimal way to do this would be:

```rust
static WASMER0_TRAP_HANDLER_SETUP: std::sync::Once = std::sync::Once::new();
```

* no need to depend on external library that way
* no need to gate the import.

Reply:
Sounds good. Replace with std Once now. However install the handler only in test does not bring correct behavior in runner, and install wasmer0 handerl indrectly via invoke `error_cases::trap_contract()` is not very good, therefore I'll move this to a direct install and to vm-runner.